zebra-state/src/config.rs

```diff
@@ -87,7 +96,7 @@ impl Default for Config {
 
         Self {
             cache_dir,
-            memory_cache_bytes: 512 * 1024 * 1024,
+            memory_cache_bytes: 50_000_000,
```

Comment:
This doesn't correspond to an actual in-memory size, so there's no reason to have an exact power of two size.

Reply:
It's also much easier for users to understand and tweak a default that ends with lots of zeroes.